src/domains/shared-app/03_influxdb.tf

```diff
@@ -0,0 +1,35 @@
+# influxdb v2
+resource "helm_release" "influxdb2" {
+  name = "pagopa-influxdb2"
+
+  repository = "https://helm.influxdata.com/"
+  chart      = "influxdb2"
+  version    = var.influxdb2_helm_version
+  namespace  = kubernetes_namespace.namespace.metadata[0].name
+
+  set {
+    name  = "service.type"
+    value = "ClusterIP"
+  }
+
```

Comment:
it's a default type probably we can remove it

Reply:
👍